sapp/ui/frontend/src/Filter.js

```diff
@@ -920,6 +921,21 @@ export function loadFilter(): ?FilterDescription {
   return null;
 }
 
+export function clearFilter(
+  refetch: any = false,
+  setCurrentFilter: any = false,
+  currentFilter: FilterDescrition = emptyFilter,
+) {
+  if(!refetch) {
+    window.sessionStorage.removeItem('filter');
+  } else  {
+    setCurrentFilter(emptyFilter);
+    // Hacky solution to the hacky clearAndRefetch function
```

Comment:
For future folks looking at this code, could you replace this comment to include why `refetch` needs to be called twice?

Reply:
Sure